cfme/tests/automate/test_method.py

```diff
@@ -348,3 +350,99 @@ def test_send_email_method(smtp_test, klass):
     # TODO(GH-8820): This issue should be fixed to check mails sent to person in 'cc' and 'bcc'
     # Check whether the mail sent via automate method really arrives
     wait_for(lambda: len(smtp_test.get_emails(to_address=mail_to)) > 0, num_sec=60, delay=10)
+
+
+@pytest.fixture(scope="module")
+def generic_definition(appliance):
```

Comment:
Why not call this a `generic_object_definition`? I think that would be a bit more descriptive.